megamek/src/megamek/client/bot/princess/BasicPathRanker.java

```diff
@@ -372,13 +374,13 @@ EntityEvaluationResponse evaluateMovedEnemy(Entity enemy, MovePath path,
 
         EntityEvaluationResponse returnResponse = new EntityEvaluationResponse();
 
-        int distance = enemy.getPosition().distance(path.getFinalCoords());
-
+        int distance = enemy.getPosition().distance(path.getFinalCoords());                
+        
         // How much damage can they do to me?
         double theirDamagePotential = calculateDamagePotential(enemy,
-                                                               null,
+                                                               new EntityState(enemy),
```

Comment:
This innocuous looking set of changes makes it so the bot consider's its own units' end point for damage potential purposes, rather than where it started. Which might make a difference.